pkgs/applications/version-management/git-and-tools/git-team/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, buildGoModule, fetchFromGitHub, installShellFiles }:
+buildGoModule rec {
+  pname = "git-team";
+  version = "1.7.0";
+
+  src = fetchFromGitHub {
+    owner = "hekmekk";
+    repo = "git-team";
+    rev = "v${version}";
+    sha256 = "0nl5j64b61jw4bkf29y51svjbndmqqrqx96yaip4vjzj2dx9ywm4";
+  };
+
+  vendorSha256 = "sha256-xJMWPDuqoNtCCUnKuUvwlYztyrej1uZttC0NsDvYnXI=";
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  postInstall = ''
+    go run main.go --generate-man-page > $out/${pname}.1
+    installManPage $out/${pname}.1
+
+    # Currently only bash completions are provided
+    go run main.go completion bash > $out/${pname}.bash
+    installShellCompletion --bash --name ${pname}.bash $out/${pname}.bash
```

Comment:
```suggestion
    go run main.go completion bash > $out/${pname}.bash
    installShellCompletion --cmd git-team --bash <($out/bin/git-team completion bash)
```

Reply:
You probably also meant rm line 22, right? Otherwise it doesn't make sense to me.

Reply:
> You probably also meant rm line 22, right? Otherwise it doesn't make sense to me.

Yeah.

We don't want to create anything temporary under `$out/`. Also install shell completion like that is the nice and short way and also allows easily adding fish and zsh later.

Reply:
Fine. I don't mind. Updated.
Where can I find more about the semantic regarding $out ?
Because I can't follow your argument why putting it in $out is a no-no.

Reply:
`$out` is basically what later will be the package contents. If you place temporary files there and leave them then will end up in the final package.